tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Properties/InterceptingProjectProperties/ApplicationPropertyPage/CompoundTargetFrameworkValueProviderTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    public class CompoundTargetFrameworkValueProviderTests
+    {
+        [Fact]
+        public async Task WhenChangingTargetFramework_PlatformPropertiesAreDeleted()
```

Comment:
Can we also add a test where the TF is newer than .NET 5, to validate that the code no longer modifies these properties in that case?